docs/content/en/open_source/api-v2-docs.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "DefectDojo API v2"
+description: "DefectDojo's API lets you automate tasks, e.g. uploading scan reports in CI/CD pipelines."
+draft: false
+weight: 2
+---
+
+
+
+
+DefectDojo\'s API is created using [Django Rest
+Framework](http://www.django-rest-framework.org/). The documentation of
+each endpoint is available within each DefectDojo installation at
+[`/api/v2/doc/`](https://demo.defectdojo.org/api/v2/) and can be accessed by choosing the API v2
+Docs link on the user drop down menu in the header. 
+
+![image](../../images/api_v2_1.png)
+
+The documentation is generated using [drf-spectacular](https://drf-spectacular.readthedocs.io/) at [`/api/v2/oa3/swagger-ui/`](https://demo.defectdojo.org/api/v2/oa3/swagger-ui/), and is
+interactive. On the top of API v2 docs is a link that generates an OpenAPI v3 spec.
+
+To interact with the documentation, a valid Authorization header value
+is needed. Visit the `/api/key-v2` view to generate your
+API Key (`Token <api_key>`) and copy the header value provided.
+
+![image](../../images/api_v2_2.png)
+
+Each section allows you to make calls to the API and view the Request
+URL, Response Body, Response Code and Response Headers.
+
+![image](../../images/api_v2_3.png)
+
+If you're logged in to the Defect Dojo web UI, you do not need to provide the authorization token.
+
+## Authentication
+
+The API uses header authentication with API key. The format of the
+header should be: :
+
+    Authorization: Token <api.key>
+
+For example: :
+
+    Authorization: Token c8572a5adf107a693aa6c72584da31f4d1f1dcff
+
+### Alternative authentication method
+
+If you use [an alternative authentication method](../social-authentication/) for users, you may want to disable DefectDojo API tokens because it could bypass your authentication concept. \
```

Comment:
Should this be updated for the new path?

```suggestion
If you use [an alternative authentication method](/en/open_source/archived_docs/integrations/social-authentication/) for users, you may want to disable DefectDojo API tokens because it could bypass your authentication concept. \
```